prql-compiler/src/prql.pest

```diff
@@ -15,7 +15,7 @@ func_def_param = { ident_part ~ type_def? ~ (":" ~ expr)? }
 type_def = { "<" ~ type_term ~ ( "|" ~ type_term)* ~ ">" }
 type_term = { ident_part ~ type_def? }
 
-table_def = { "table" ~ ident_part ~ "=" ~ nested_pipeline ~ ( NEWLINE+ | &EOI ) }
```

Comment:
It really was that easy. Nice.

Reply:
Pest is actually very good at this — would be more difficult in a grammar that restricted ambiguity, like Lezer